contracts/deploy/Create3.sol

```diff
@@ -21,13 +21,28 @@ contract CreateDeployer {
     }
 }
 
+library AddressUtils {
```

Comment:
@deanamiel could you please move this into its own file under the `utils` folder? So we can also use it in other places as well. Let's call it `ContractAddress` to match the convention we have for other utils.